packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -1201,8 +1206,10 @@ class StackViewport extends Viewport implements IStackViewport, IImagesLoader {
   }
 
   private setRotationGPU(rotation: number): void {
+    const panFit = this.getPan(this.fitToCanvasCamera);
```

Comment:
can you please add a comment on why all this pan magic is required for rotation?